msm/src/lookups.rs

```diff
@@ -14,14 +14,14 @@ pub enum LookupTableIDs {
     /// Custom lookup table
     /// The index of the table is used as the ID, padded with the number of
     /// built-in tables.
-    Custom(usize),
+    Custom(u32),
```

Comment:
What's the reason that motivates you to change it to u32?

Reply:
To use `Fp::from(id)` :)

Reply:
And I think u32 is way enough :)